app/assets/scss/app.scss

```diff
@@ -0,0 +1,5 @@
+$govuk-show-breakpoints: true;
+
+@import "../../../src/govuk-frontend/all.scss";
```

Comment:
Defo can't reference this any other way? Could be fragile

Reply:
I was having a look at this and we might be able to simplify this.

Reply:
Should be fixed now

Reply:
Nice 👍